hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -148,10 +148,11 @@ public AppendResult appendBlocks(List<HoodieLogBlock> blocks) throws IOException
     HoodieLogFormat.LogFormatVersion currentLogFormatVersion =
         new HoodieLogFormatVersion(HoodieLogFormat.CURRENT_VERSION);
 
-    FSDataOutputStream outputStream = getOutputStream();
-    long startPos = outputStream.getPos();
+    FSDataOutputStream originalOutputStream = getOutputStream();
```

Comment:
Can we give some tests for it.

Reply:
Ummm. Yes we could have a test to write a huge log block to have a check. But it may affect UT performance a lot. Not sure if an UT relevant to such a reworking of existing logic is necessary.  Anyway, I'm glad to add a UT if it is compulsory.